test/powershell/Modules/Microsoft.PowerShell.Management/Copy.Item.Tests.ps1

```diff
@@ -1,6 +1,19 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+Describe "Validate Copy-Item locally" -Tags "CI" {
+    It "Copy-Item has non-terminating error if destination is in use" -Skip:(!$IsWindows) {
+        Copy-Item -Path $env:windir\system32\cmd.exe -Destination TestDrive:\
```

Comment:
Can we use `pwsh` and make this test cross platform?

Reply:
Non-Windows doesn't have a file open handle problem and will let you overwrite a file that's in use